CMakeLists.txt

```diff
@@ -125,7 +130,7 @@ list(FILTER UTILS_HEADER EXCLUDE REGEX cgal*)
 add_library(utils STATIC ${UTILS_SRCS} ${UTILS_HEADER})
 set_property(TARGET utils PROPERTY CXX_STANDARD 14)
 set_property(TARGET utils PROPERTY CXX_STANDARD_REQUIRED ON)
-target_link_libraries(utils igl::core igl::opengl igl::cgal igl::triangle spdlog Qt5::Core Qt5::Widgets spdlog)
+target_link_libraries(utils igl::core igl::opengl igl::cgal igl::triangle spdlog Qt5::Core Qt5::Widgets spdlog glfw ${GLFW_LIBRARIES})
```

Comment:
I don't see the variable `GLFW_LIBRARIES` set anywhere. I think we can remove it.